config/config.txt

```diff
@@ -275,7 +275,7 @@ CHECK_RANDOMIZER
 # FORUMURL http://tgstation13.org/phpBB/index.php
 
 ## Wiki address
-# WIKIURL http://www.tgstation13.org/wiki
+# WIKIURL https://wiki.skyrat13.tk/w/index.php
```

Comment:
don't change this, 'tis config

Reply:
Indeed, since it's commented out, it doesn't actually need to be changed, here.